test/test-applications/integrations/TestApplication.Wcf.Server.NetFramework/Program.cs

```diff
@@ -17,26 +17,30 @@
 using System;
 using System.ServiceModel;
 using System.Threading;
+using System.Threading.Tasks;
 
 namespace TestApplication.Wcf.Server.NetFramework;
 
 internal static class Program
 {
-    public static void Main()
+    public static async Task Main()
     {
         try
         {
             try
```

Comment:
why there is a double-try? I see no benefits of the outer try

Reply:
Hmm, maybe it did at some point. I think that was leftover after @pjanotti was testing it. I will remove it.